src/main/java/interface_adapter/edit_assignment/EditAssignmentViewModel.java

```diff
@@ -1,12 +1,11 @@
-package src.main.java.interface_adapter.edit_assignment;
+package interface_adapter.edit_assignment;
 
 import interface_adapter.ViewModel;
-
-public class EditAssignmentViewModel extends ViewModel<EditAssignmentState> {
-
+//  Temporary fix but would be more efficient without the hefty import
+public class EditAssignmentViewModel extends ViewModel<src.main.java.interface_adapter.edit_assignment.EditAssignmentState> {
     public EditAssignmentViewModel() {
-        super("EditAssignment");
-        setState(new EditAssignmentState());
+        super("Edit Assignment");
+        setState(new src.main.java.interface_adapter.edit_assignment.EditAssignmentState());
```

Comment:
Here do you need "rc.main.java.interface_adapter.edit_assignment..."? Does it not work when its just EditAssignmentState()?

Reply:
I tried but it wasn't working any other way for some reason /: I've kept it there temporarily, will definitely try cracking through it again.